src/xdp/trn_transit_scaled_endpoint.c

```diff
@@ -1,664 +0,0 @@
-// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Leave this file in, we need it

Reply:
I think we can probably add it back later as this scaled endpoint implementation doesn't quite match with the Zeta's current scaled endpoint design